.github/workflows/test-unreleased.yml

```diff
@@ -14,34 +14,50 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        entry:
-          - { opensearch_ref: '1.x' }
-          - { opensearch_ref: '2.x' }
-          - { opensearch_ref: '2.0' }
-          - { opensearch_ref: 'main' }
+        opensearch_ref: [ '1.x', '2.x', '2.0', 'main' ]
+        ruby_version: 3.1
```

Comment:
Do we want to make this `3.2` or add `3.2` to the matrix now that it is released?

Reply:
I don't think it matters. The goal of this workflow is to test the unreleased OS. We have other workflows that test Ruby 3.2